### ISSUE TITLE
Wer increases but the training and validation loss decreases

### ISSUE THREAD

**Issue by mm0177** [huggingface/community-events#202]
I have been trying to fine tune the whisper base on an english dataset only to improve the transciption of the model as the wer seemed to be high without fine tuning. I have a total of 560 audio files to fine tune it . 
![Base model](https://github.com/huggingface/community-events/assets/113229869/4690caf4-03ab-479f-9539-0b9260e6db62)
The above picture shows the base model performnace , there was actually very high difference in the train and validation loss os i used lora optimizer. but after using that you can see the abouve issue in the image which i am facing 

![Screenshot 2024-07-03 015243](https://github.com/huggingface/community-events/assets/113229869/eeae694a-2c50-43fa-b7c4-7454688a0cc8)
When i used large-v2 model the above image as you can see , i am facing that. In this also i used the lora optimizer. 

I have been stuck in this what is the issue and how can i solve this 

Thank you in advance!
@Vaibhavs10 @sanchit-gandhi
